M2/Macaulay2/c/README

```diff
@@ -266,7 +266,7 @@ Expressions:
 	if a then b
 	until a do b
 	while a do b			evaluates b as long as a is true
-	while a list b			produces a list of the values of b, as long as a is true
+	while a list b			produces a list of the values of b, as long as a is true (not implemented!?)
```

Comment:
I think this was meant to be `while a do provide b`.

Reply:
There's also [`while p list x do z`](https://macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_while.html) in top-level m2 [which you probably already know], which does create a list of `x` values.